dom/package.json

```diff
@@ -33,54 +33,54 @@
   "module": "lib/es6/index.js",
   "typings": "lib/cjs/index.d.ts",
   "types": "lib/cjs/index.d.ts",
+  "scripts": {
+    "build": "pnpm run build-cjs && pnpm run build-es6",
+    "build-cjs": "tsc --module commonjs --outDir ./lib/cjs",
+    "build-es6": "tsc --module es6 --outDir ./lib/es6",
+    "test-node": "mocha test/node/*.ts --require ts-node/register --exit",
+    "test-browser": "karma start",
+    "test": "pnpm run test-node && pnpm run test-browser",
+    "test-ci": "CI=true pnpm test",
+    "test-watch": "WATCH=true pnpm test",
+    "test-live": "LIVE=true karma start --watch"
+  },
   "dependencies": {
     "@cycle/run": "*",
-    "es6-map": "^0.1.4",
-    "snabbdom": "0.7.0",
+    "es6-map": "^0.1.5",
+    "rxjs-compat": "^6.2.2",
```

Comment:
Hmm, `rxjs-compat` as a *dependency* of /dom? Shouldn't it be devDependency?

Reply:
Yes it should